var/spack/repos/builtin/packages/citcoms/package.py

```diff
@@ -27,7 +27,6 @@ class Citcoms(AutotoolsPackage):
     # Required dependencies
     depends_on('mpi')
     depends_on('zlib')
-    depends_on('python@:2', type='run')  # needed for post-processing scripts
     depends_on('automake', when='@master', type='build')
```

Comment:
Why does this work and not break?

Reply:
The Python dependency was optional and only needed if you want to manually run some post-processing scripts. I think in this case it's better to remove the dependency than remove the package. If I ever need the post-processing scripts in the future I'll probably just submit a PR to port them to Python 3.